test/functional/answers_controller_test.rb

```diff
@@ -9,11 +9,15 @@ def setup
   test "should get create if user is logged in" do
     UserSession.create(rusers(:bob))
     node = node(:question)
-    assert_difference 'Answer.count' do
-      xhr :post, :create,
-                 nid: node.nid,
-                 body: "Sample answer"
+    count = ActionMailer::Base.deliveries.length
+    assert_difference 'ActionMailer::Base.deliveries.size', 1 do
+      assert_difference 'Answer.count' do
+        xhr :post, :create,
+                   nid: node.nid,
+                   body: "Sample answer"
+      end
     end
+    assert ActionMailer::Base.deliveries.collect(&:to).includes?('author@email.com')
```

Comment:
Can you refer to the fixture property rather than hard coding the address here? So "node(: question).author.email", I think?